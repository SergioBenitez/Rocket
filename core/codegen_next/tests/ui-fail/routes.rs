#![feature(plugin, decl_macro, proc_macro_non_items)]

#[macro_use] extern crate rocket;

fn main() {
    let _ = routes![a b];
    //~^ ERROR expected
}
