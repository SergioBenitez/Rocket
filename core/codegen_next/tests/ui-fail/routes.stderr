error: expected `,` or `::` or end of macro invocation
 --> $DIR/routes.rs:6:23
  |
6 |     let _ = routes![a b];
  |                       ^

error: aborting due to previous error

